streamwebs_frontend/streamwebs/templates/streamwebs/sites.html

```diff
@@ -62,5 +62,9 @@
             {% endfor %}
             </ul>
         </div>
+        <br>
+        <div id="create-site">
+            <a href="{% url 'streamwebs:create_site' %}" class="btn teal darken-4">CREATE_SITE</a>
```

Comment:
The name for the url.py is create_site when I pulled it, do I still change it or leave it as underscore?